app/views/users/notifications.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="page-header">
+  <h1><%= t 'view.user.notifications.settings' %></h1>
+</div>
+<%= form_for(@user, url: user_path(@user), method: :patch) do |f| %>
+  <div class="form-group">
+    <div class="col-lg-offset-2 col-lg-10">
+      <%= f.check_box :team_notifications_enabled %>
```

Comment:
My personal preference would be to have the `filled-in` class on these checkboxes which would make them look like this:

<img width="324" alt="ohne titel 2" src="https://user-images.githubusercontent.com/807665/35525313-6ff70788-0524-11e8-9ea9-f4aea9ff3058.png">